readme.md

```diff
@@ -29,6 +29,11 @@ Appender configuration sample
 		-->
 		<connectionStringName value="mongo-log4net" />
 		<!-- 
+		If set, a TTL (Time To Live) index will be created on the Timestamp field.  
+		Records older than this value will be deleted.
+		-->		
+		<expireAfterSeconds value='5' />

```

Comment:
just to be consistent with quotes in readme.md, please change value='5' to value="5" and I'll merge this. Thanks